lib/ui.js

```diff
@@ -394,9 +398,23 @@ exports.clearSettingsMap = function(aWin) {
 };
 
 /**
+ * Handle firstrun page
+ */
+pageMod.PageMod({
+  include: data.url("firstRun.html*"),
```

Comment:
Nope! It covers the case where there is an anchor tag after the resource, such as firstRun.html#slideshow

Otherwise the script doesn't load when you click the link in the popup.